tests/Hedgehog.Tests/Hedgehog.Tests.fsproj

```diff
@@ -33,29 +33,6 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Release\Hedgehog.Tests.XML</DocumentationFile>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="mscorlib" />
-    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Numerics" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AssemblyInfo.fs" />
-    <Compile Include="GenTests.fs" />
-    <Compile Include="ShrinkTests.fs" />
-    <Compile Include="MinimalTests.fs" />
-    <Content Include="App.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\Hedgehog\Hedgehog.fsproj">
-      <Name>Hedgehog</Name>
-      <Project>{b338b8c4-1c98-4f42-9c45-56bd7ef2ad0a}</Project>
-      <Private>True</Private>
-    </ProjectReference>
-  </ItemGroup>
   <PropertyGroup>
```

Comment:
I guess these items where moved automatically? It's hard to spot if there are differences here on not 😕